packages/components/tooltip/src/tooltip.ts

```diff
@@ -88,13 +30,28 @@ export const useTooltipProps = buildProps({
   },
 })
 
-export type ElTooltipContentProps = ExtractPropTypes<
-  typeof useTooltipContentProps
->
-
-export type ElTooltipTriggerProps = ExtractPropTypes<
-  typeof useTooltipTriggerProps
->
+export const createModelToggleComposableResult = createModelToggleComposable(
+  'visible' as const
+)
+
+export const TooltipProps = {
+  ...usePopperProps,
+  ...createModelToggleComposableResult.useModelToggleProps,
+  ...useTooltipContentProps,
+  ...useTooltipTriggerProps,
+  ...usePopperArrowProps,
+  ...useTooltipProps,
+}
```

Comment:
**Suggestion**
It is recommended to use the lower camel case for variable definitions and the upper camel case for type definitions.

```ts
export const useTooltipProps = buildProps({
  ...usePopperProps,
  ...useModelToggleProps,
  ...useTooltipContentProps,
  ...useTooltipTriggerProps,
  ...usePopperArrowProps,
  openDelay: {
    type: Number,
  },
  visibleArrow: {
    type: Boolean,
    default: undefined,
  },
  showArrow: {
    type: Boolean,
    default: true,
  },
})
export type ElTooltipProps = ExtractPropTypes<typeof useTooltipProps>
```

`emits` too.